src/listener.cpp

```diff
@@ -19,7 +19,7 @@
 #include <std_msgs/msg/string.hpp>
 
 
-void chatterCallback(const std_msgs::msg::String::ConstSharedPtr & msg)
+void chatterCallback(const std_msgs::msg::String::ConstSharedPtr msg)
```

Comment:
This PR only removes the reference. I would rather not change the smart pointer type here.